copyright

```diff
@@ -490,6 +490,24 @@ Copyright: Becca Tommaso (tommasobecca03@gmail.com)
 License: CC-BY-SA-4.0
 Comment: Derived from works by Michael Zahniser (under the same license).
 
+Files
+ sounds/pincer*
+Copyright: Nomadic Volcano (nomadicvolcano@gmail.com)
+License: GPL 2.0
+Comment: Nomadic Volcano modified these from Battle for Wesnoth (https://www.wesnoth.org) sounds, which are copyright David White <dave@whitevine.net> under GPL 2.0
```

Comment:
```suggestion
Comment: Nomadic Volcano modified these from Battle for Wesnoth (https://www.wesnoth.org) sounds, which are copyright David White <dave@whitevine.net> under GPL 2 or later.
```
cf. The [copyright file](https://github.com/wesnoth/wesnoth/blob/master/copyright) in the current version of The Buttle for Wesnoth.